services/motion/builtin/builtin_test.go

```diff
@@ -183,18 +183,21 @@ func TestMoveSingleComponent(t *testing.T) {
 	ms = setupMotionServiceFromConfig(t, "../data/moving_arm.json")
 
 	t.Run("succeeds with supplemental info in world state", func(t *testing.T) {
+		homePose, err := ms.GetPose(context.Background(), arm.Named("pieceArm"), "", nil, nil)
+		test.That(t, err, test.ShouldBeNil)
+
 		testPose := spatialmath.NewPose(
-			r3.Vector{X: 1., Y: 2., Z: 3.},
-			&spatialmath.R4AA{Theta: math.Pi / 2, RX: 0., RY: 1., RZ: 0.},
+			r3.Vector{homePose.Pose().Point().X + 20, homePose.Pose().Point().Y, homePose.Pose().Point().Z},
```

Comment:
new inputs are modeled after:
https://github.com/viamrobotics/rdk/blob/main/components/arm/arm_test.go#L455

Reply:
our home position is:
```
startPose x:-817.2 y:-232.9 z:62.8 o_x:1.5700924586837752e-16 o_y:-1 o_z:1.7938038903913487e-16 theta:89.99999999999999
```

since our goal is wrt another frame, its converted point is:
```
builtin/builtin.go:165  converted goal pose "x:-796.2 y:-231.9 z:63.8 o_z:1"
```